tests/integration/scheduler/test_eval.py

```diff
@@ -215,7 +217,7 @@ def test_eval_once_loop_interval(self):
         self.schedule.opts['loop_interval'] = LOOP_INTERVAL
 
         # Run the job at the right plus LOOP_INTERVAL
-        run_time = int(time.mktime(dateutil_parser.parse('12/13/2017 1:00:{0}pm'.format(LOOP_INTERVAL)).timetuple()))
+        run_time = dateutil_parser.parse('12/13/2017 1:00:{0}pm'.format(LOOP_INTERVAL))
```

Comment:
Zero pad the seconds? Or will it still get passed?

Reply:
Going to swap this out for a datetime.timedelta which is cleaner.